cbrkit/loaders.py

```diff
@@ -1,3 +1,18 @@
+"""
+To manually use Pydantic with CBRkit to validate your case base, you can use an appropriate 
+Pydantic model instead of the CBRkit loaders (see example below). 
+Alternatively, the dataframe, path, file and folder accept an optional validation_model argument
+to validate the Casebase entries.
```

Comment:
The description should be updated for the new `cbrkit.loaders.validate` function.